nerium-k8s.yml

```diff
@@ -0,0 +1,45 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: nerium
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: nerium
+    spec:
+      containers:
+      - name: nerium
+        image: us.gcr.io/lexical-cider-93918/nerium:latest
+        ports:
+        - containerPort: 8081
+        volumeMounts:
+        - name: dotenv
+          mountPath: /dotenv
+        - mountPath: /etc/ssl/certs
+          name: ca-certificates
+        securityContext:
+          capabilities:
+            add:
+            - SYS_ADMIN
+          privileged: true
+        lifecycle:
+          postStart:
+            exec:
+              command: ["gcsfuse", "nerium-sql", "/app/query_files"]
+          preStop:
+            exec:
+              command: ["fusermount", "-u", "/app/sqls"]
```

Comment:
Should this be `/app/query_files` instead?

Reply:
Yup